### DISCUSSION TITLE
What icon packed is used by Taiga UI?

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9241] [Q&A] [answered]
I cannot find under Icons information what icon packed is used by Taiga UI? if it is not google icons then - where to find full icon list?
I have found it here https://github.com/taiga-family/taiga-ui/blob/8e78f6f63d26cbd28a068cd5b55e6019fd13c3d4/projects/demo/used-icons.ts#L9 but:

1. whole list with look would be appriaciated
2. example of how to use external items

**Comment by splincode** [comment, marked as answer]
<img width="577" alt="image" src="https://github.com/user-attachments/assets/c94779e4-5e31-4ff9-9d7f-aee5e035ce84">

https://lucide.dev/icons/

**Comment by vytautas-pranskunas-** [comment]
![Screenshot 2024-09-27 at 14 53 13](https://github.com/user-attachments/assets/ac1efb44-ca96-4e06-aadf-6e6832c5d8c3)
 this is what i see. Are you using different documentation?

**Reply by splincode** [reply]
<img width="261" alt="image" src="https://github.com/user-attachments/assets/31828f7c-316a-41e8-bf39-815523bbe861">

**Reply by waterplea** [reply]
The page you are looking at is Icons **component**. It can show any icon and color it with CSS. It answers your second question how to use external icons with various examples. As for all the icons there are links @splincode suggested. I'll link them all together with See also so it's easier to find next time.

![image](https://github.com/user-attachments/assets/ab21cc24-0b73-49f8-91d4-d9862a05c073)

**Comment by vytautas-pranskunas-** [comment]
Am i looking at wrong documentation because i do not see expander next to icons side menu

![Screenshot 2024-09-27 at 17 17 40](https://github.com/user-attachments/assets/261d5049-663f-4083-9833-b8e562128ae3)

**Comment by vytautas-pranskunas-** [comment]
Ok, that was confusing. I have found it not under components but on very top there is separate Icons section.
It says that it supports Material icons out of the box: "Taiga UI supports [Material Icons. ](https://fonts.google.com/icons?icon.set=Material+Icons)You can browse all the icons from this pack and find the one that suits you [here. ](https://marella.me/material-design-icons/demo/font/)You can replace icons used inside components as well through DI options, see example below."

but when i try to use icons i got in console such error (icon taken from documentation example)

`GET http://localhost:4200/assets/taiga-ui/icons/material/filled/favorite.svg 404 (Not Found)`

**Reply by waterplea** [reply]
Did you do this?
https://taiga-ui.dev/icons/material/Setup

**Comment by vytautas-pranskunas-** [comment]
Oh, i need to get used to documentation structure. I really did not see
that tab because for some menu items it is there for some not. Not very
intuitive at least for me. Thank you
